package.json

```diff
@@ -198,6 +203,27 @@
                     "type": "string",
                     "default": "...:*",
                     "description": "A [query language expression](https://bazel.build/query/language) which determines the packages displayed in the workspace tree and quick picker. The default inspects the entire workspace, but you could narrow it. For example: `//part/you/want/...:*`"
+                },
+                "bazel.lsp.enabled": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Enable the LSP (experimental)"
```

Comment:
Could we add a note here to remind that `lsp.command` needs to be set in order for it to work?

We can also imply it to be true when `lsp.command` is set.

Reply:
`lsp.command` defaults to the executable name that cargo will install it under (`starlark`), so setting this shouldn't be necessary unless you're doing something strange.

Reply:
I see. I also meant to ask that should we remove the default `starlark` from `lsp.command`?

Instead of having these scenarios:

1.
```
enabled: False
command: "starlark" // no-op
```
2.
```
enabled: True
command: "starlark"
```
, we'll just have
1.
```
command: undefined // disabled
```
2.
```
command: "starlark"
```

Just one less configuration to simply it a little bit in case users don't see the README before seeing configurations. By forcing users to specify `starlark` explicitly, they're more aware that they need to build/install the `starlark` binary in the first place. If not, `starlark` stays undefined until `cargo` builds it.

Reply:
Personally I think having a single config option mean two different things is a bit confusing, and checking a checkbox is easier than writing "starlark". I think the case where the user hasn't installed the binary should be handled with an informative error message, but this is pretty experimental so I'm hesitant to add that sort of polish yet.

Reply:
> Personally I think having a single config option mean two different things is a bit confusing

It doesn't necessarily mean two different things. It can be interpreted as

```
If lsp command is specified, use it to start a lsp client.
Otherwise, don't
```

This is so that we don't need to add the informative error message. The config is then self-explanatory.

Anyways, this is just my opinion. I checked out your branch and it worked great! Thanks!